src/main.rs

```diff
@@ -9,9 +9,15 @@
 // specific language governing permissions and limitations under the License.
 
 extern crate clap;
+#[macro_use] extern crate nickel;
+
+use nickel::{Nickel, HttpRouter};
 
 use clap::{App, Arg, SubCommand, AppSettings};
 
+use std::u16;
+use std::str::FromStr;
```

Comment:
https://aturon.github.io/style/imports.html

Prefer

```use std::foo::{Bar, Baz};```

over

```use std::foo::*;```

because wildcard imports can change over time, introducing new name collisions.

Another approach that I'm starting to favor is:

```
use foo::edn;
…

fn bar(keyword: edn::Keyword) -> () {
```

Using explicit qualification can introduce clarity and avoids the decision about precisely what you need to `use`.